pkg/facade/computerFacade.go

```diff
@@ -0,0 +1,43 @@
+package facade
+
+import (
+	"fmt"
+	"strings"
+)
+
+// computerFacade implements facade
+type computerFacade struct {
```

Comment:
package facade

import (
	"fmt"
	"strings"
)

type cpu interface {
	// cpu used functions in current module
}

type memory interface {
	// memory used functions in current module
}

type hardDrive interface {
	// hardDrive used functions in current module
}

type ComputerFacade interface {
	MakeSound() string
	Start() string
}

// computerFacade implements facade
type computerFacade struct {
	sound     string
	cpu       cpu
	memory    memory
	hardDrive hardDrive
}

func (f *computerFacade) MakeSound() string {
	return fmt.Sprintf("...%v...", f.sound)
}

// Start returns how computer will start.
func (f *computerFacade) Start() string {
	result := []string{
		makeComponentWorkingSound(f),
		f.cpu.Freeze(),
		makeComponentWorkingSound(f.cpu),
		f.memory.Load(f.memory.position, f.hardDrive.Read()),
		makeComponentWorkingSound(f.memory),
		makeComponentWorkingSound(f.hardDrive),
		f.cpu.Jump(f.memory.position),
		f.cpu.Execute(),
	}
	return strings.Join(result, "\n")
}

// NewComputerFacade creates computerFacade
func NewComputerFacade(
	position string, 
	lba, size int,
	cpu       cpu,
	memory    memory,
	hardDrive hardDrive,
) ComputerFacade {
	return &computerFacade{
		sound:     "sounds of starting computer",
		cpu:       cpu,
		memory:    memory,
		hardDrive: hardDrive,
	}
}